Source/Mosa.Tool.Launcher/MainWindow.axaml.cs

```diff
@@ -275,23 +268,73 @@ private void UpdateSettings()
+		settings.SetValue("Image.FileSystem.RootInclude", IncDirTxt.Text);
+	}
+
 	private void SetDefaultSettings()
 	{
+		settings.SetValue("Compiler.BaseAddress", 0x00400000);
 		settings.SetValue("Compiler.Binary", true);
-		settings.SetValue("Compiler.Multithreading.MaxThreads", Environment.ProcessorCount);
+		settings.SetValue("Compiler.MethodScanner", false);
+		settings.SetValue("Compiler.Multithreading", true);
+		settings.SetValue("Compiler.Platform", "x86");
 		settings.SetValue("Compiler.TraceLevel", 0);
+		settings.SetValue("CompilerDebug.DebugFile", string.Empty);
+		settings.SetValue("CompilerDebug.AsmFile", string.Empty);
+		settings.SetValue("CompilerDebug.MapFile", string.Empty);
+		settings.SetValue("CompilerDebug.NasmFile", string.Empty);
+		settings.SetValue("CompilerDebug.InlineFile", string.Empty);
+		settings.SetValue("Optimizations.Basic", true);
+		settings.SetValue("Optimizations.BitTracker", true);
+		settings.SetValue("Optimizations.Inline", true);
 		settings.SetValue("Optimizations.Inline.AggressiveMaximum", 24);
+		settings.SetValue("Optimizations.Inline.Explicit", true);
 		settings.SetValue("Optimizations.Inline.Maximum", 12);
 		settings.SetValue("Optimizations.Basic.Window", 5);
+		settings.SetValue("Optimizations.LongExpansion", true);
+		settings.SetValue("Optimizations.LoopInvariantCodeMotion", true);
+		settings.SetValue("Optimizations.Platform", true);
+		settings.SetValue("Optimizations.SCCP", true);
+		settings.SetValue("Optimizations.Devirtualization", true);
+		settings.SetValue("Optimizations.SSA", true);
+		settings.SetValue("Optimizations.TwoPass", true);
+		settings.SetValue("Optimizations.ValueNumbering", true);
+		settings.SetValue("Image.Folder", Path.Combine(Path.GetTempPath(), "MOSA"));
+		settings.SetValue("Image.Format", "IMG");
+		settings.SetValue("Image.FileSystem", "FAT16");
 		settings.SetValue("Image.ImageFile", "%DEFAULT%");
+		settings.SetValue("Image.Firmware", "bios");
 		settings.SetValue("Multiboot.Version", "v1");
+		settings.SetValue("Multiboot.Video", false);
+		settings.SetValue("Multiboot.Video.Width", 640);
+		settings.SetValue("Multiboot.Video.Height", 480);
+		settings.SetValue("Multiboot.Video.Depth", 32);
+		settings.SetValue("Emulator", "Qemu");
+		settings.SetValue("Emulator.Memory", 128);
+		settings.SetValue("Emulator.Cores", 1);
+		settings.SetValue("Emulator.Serial", "none");
 		settings.SetValue("Emulator.Serial.Host", "127.0.0.1");
 		settings.SetValue("Emulator.Serial.Port", new Random().Next(11111, 22222));
 		settings.SetValue("Emulator.Serial.Pipe", "MOSA");
 		settings.SetValue("Emulator.Display", true);
-		settings.SetValue("Launcher.Start", false);
+		settings.SetValue("Launcher.Start", true);
 		settings.SetValue("Launcher.Launch", true);
+		settings.SetValue("Launcher.Exit", true);
+		settings.SetValue("Launcher.PlugKorlib", true);
 		settings.SetValue("Launcher.HuntForCorLib", true);
+		settings.SetValue("Linker.Drawf", false);
```

Comment:
is this a typo?

Reply:
I copied them from the console launcher. It doesn't seem to be affecting the launcher though?